package.json

```diff
@@ -12,11 +12,13 @@
     "@babel/helper-module-imports": "7.0.0",
     "@babel/preset-env": "7.3.1",
     "@babel/preset-typescript": "7.9.0",
+    "@lodder/grunt-postcss": "^3.1.1",
```

Comment:
grunt-postcss fell way behind and is now a readonly repository. Someone forked it and updated it to work with postcss 8.